GHCommentAuthorAssociation MANNEQUIN missing

**Describe the bug**
In the enum GHCommentAuthorAssociation there can come another value that can not be parsed: MANNEQUIN 
Also see: https://docs.github.com/en/graphql/reference/enums#commentauthorassociation

```
Caused by: com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `org.kohsuke.github.GHCommentAuthorAssociation` from String "MANNEQUIN": not one of the values accepted for Enum class: [OWNER, FIRST_TIMER, NONE, MEMBER, CONTRIBUTOR, COLLABORATOR, FIRST_TIME_CONTRIBUTOR]
```


**To Reproduce**
Try to get revuew comments of with an unclaimed user as author.

**Expected behavior**
No error.


**Additional context**
This value is possible for migrated repositories.

--- a/src/main/java/org/kohsuke/github/GHCommentAuthorAssociation.java
+++ b/src/main/java/org/kohsuke/github/GHCommentAuthorAssociation.java
@@ -26,7 +26,8 @@ public enum GHCommentAuthorAssociation {
     /**
      * Author is a placeholder for an unclaimed user.
      */
-    MANNEQUIN,    /**
+    MANNEQUIN,
+    /**
      * Author is a member of the organization that owns the repository.
      */
     MEMBER,

--- a/src/main/java/org/kohsuke/github/GHCommentAuthorAssociation.java
+++ b/src/main/java/org/kohsuke/github/GHCommentAuthorAssociation.java
@@ -34,5 +34,9 @@ public enum GHCommentAuthorAssociation {
     /**
      * Author is the owner of the repository.
      */
-    OWNER
+    OWNER,
+    /**
+     * Author is a placeholder for an unclaimed user.
+     */
+    MANNEQUIN
 }

--- a/src/main/java/org/kohsuke/github/GHCommentAuthorAssociation.java
+++ b/src/main/java/org/kohsuke/github/GHCommentAuthorAssociation.java
@@ -24,6 +24,9 @@ public enum GHCommentAuthorAssociation {
      */
     FIRST_TIME_CONTRIBUTOR,
     /**
+     * Author is a placeholder for an unclaimed user.
+     */
+    MANNEQUIN,    /**
      * Author is a member of the organization that owns the repository.
      */
     MEMBER,
@@ -34,9 +37,5 @@ public enum GHCommentAuthorAssociation {
     /**
      * Author is the owner of the repository.
      */
-    OWNER,
-    /**
-     * Author is a placeholder for an unclaimed user.
-     */
-    MANNEQUIN
+    OWNER
 }

--- a/src/test/java/org/kohsuke/github/EnumTest.java
+++ b/src/test/java/org/kohsuke/github/EnumTest.java
@@ -22,7 +22,7 @@ public class EnumTest extends AbstractGitHubWireMockTest {
         assertThat(GHCheckRun.Conclusion.values().length, equalTo(9));
         assertThat(GHCheckRun.Status.values().length, equalTo(4));
 
-        assertThat(GHCommentAuthorAssociation.values().length, equalTo(7));
+        assertThat(GHCommentAuthorAssociation.values().length, equalTo(8));
 
         assertThat(GHCommitState.values().length, equalTo(4));
 

